recipes/mender-client/debian-3.5.x/control

```diff
@@ -6,7 +6,9 @@ Standards-Version: 3.9.2
 Build-Depends: debhelper (>= 9)
 
 Package: mender-client
-Conflicts: mender
+Conflicts: mender, mender-auth, mender-update, mender-flash, mender-snapshot, mender-setup
+Replaces: mender-auth, mender-update, mender-flash, mender-snapshot, mender-setup
+Provides: mender-auth, mender-update, mender-flash, mender-snapshot, mender-setup
```

Comment:
I think `mender-flash` should be removed. This is a separate tool, and `autoremove` might still remove it later.

Also, mender-binary-delta uses mender-flash independently of mender-client now (for UBI), so this conflicts with that. Granted, I don't think you'll ever have mender-binary-delta on a system which also uses apt, but you can in principle.

Reply:
Makes sense.

As discussed privately, I will amend after the CI :green_circle: but merge without a new run. Then I will run again CI on the master build before publishing.